guides/welfare/raising_an_issue.md

```diff
@@ -6,17 +6,15 @@ From time to time people may have problems or concerns about their colleagues, c
 
 The first thing you can do, if you feel comfortable and able, is raise the issue directly with the person involved. This type of direct and informal approach is encouraged and we expect your colleagues who are approached responsibly to respond in a similar manner. This approach may be particularly appropriate to dealing with matters of speech and expression, where individuals may not be aware that their behaviour is causing distress.
 
-## Raise feedback via Continuous Feedback
+## Raise the issue via your mentor
 
-If you feel unable to have a conversation directly with the person involved, and they are a member of staff at Made Tech then you can encourage them to discuss the issue in their next [continuous feedback](../../team-norms/continuous_feedback.md) session by raising the feedback through that process.
-
-Continuous feedback is not anonymous but can add a layer of indirection and ensure that the issue is discussed with another member of the team. We hope this will trigger conversations amongst those in the session and we would expect proactive action to happen off the back of that session.
+If you feel unable to have a conversation directly with the person involved, and they are a member of staff at Made Tech, then you can discuss this further with your mentor. They can help you find a way to discuss this directly with the person or discuss other routes to help resolve the issue.
 
 ## Ask a colleague to intervene on your behalf
```

Comment:
Isn't this section now superseded by raising with your mentor (who will help you deliver the feedback)?

Reply:
Yes I replaced continuous feedback with being helped by your mentor (I hope I did it right)

Reply:
Yeah - I mean that covers `## Ask a colleague to intervene on your behalf` as well - you should ask your mentor for guidance handling the situation as opposed to either of these older options